src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -684,7 +684,7 @@ export function fetchExpressCareWindows() {
     });
 
     const initialState = getState();
-    const userSiteIds = selectSystemIds(initialState);
+    const userSiteIds = selectSystemIds(initialState) || [];
```

Comment:
Adding this means that instead of stack traces from the errors further down when fetching settings for these ids, we get an "Error 404" message.